src/utils/index.js

```diff
@@ -17,7 +17,7 @@ export default {
   },
   
   isObject (val) {
-    return val !== null && typeof val === 'object' && !Array.isArray(val)
+    return val instanceof Object && !Array.isArray(val)
```

Comment:
if `val` is a `object`, its obviously not an `Array`.
So the second check can be removed.

Reply:
@TantumLabs  any news?

Reply:
It's OK: https://live.bootstrap-table.com/code/wenzhixin/4670